Assignment-4/src/Card-component.js

```diff
@@ -0,0 +1,15 @@
+const CardComponent=(props)=>{

```

Comment:
You an destruture members here only..
Like -
const CardComponent=({members})=>{

This way you don't have to write extra line.

Reply:
Thanks,It is a very good idea.